### PR TITLE
added slider

### PR DESCRIPTION
**Pull request by boazmatz**

### CODE REVIEW

File: lib/db.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ function getClientConfiguration() {
         user: 'postgres',
         host: 'localhost',
         database: 'postgres',
-        port: 5432,
```
Comment (roeeyud) [inline lib/db.js]: Don't change the default port. 5432 should be you local postgres port as well

File: templates/product p with postgres and slide.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```
Comment (roeeyud) [inline templates/product p with postgres and slide.html]: Rename file to product.html

File: templates/product p with postgres and slide.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    width: 100%;
+}
+</style>
+
+<title>3d printHub</title>
+</header>
+<body style="background-color: white;">
+    </a>
+  </div>
+  <div class="container">
```
Comment (roeeyud) [inline templates/product p with postgres and slide.html]: Product template shouldn't include the head of the file.

File: templates/product p with postgres and slide.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+.product-image {
+    width: 100%;
+}
+</style>
+
+<title>3d printHub</title>
+</header>
+<body style="background-color: white;">
+    </a>
+  </div>
```
Comment (roeeyud) [inline templates/product p with postgres and slide.html]: Looks like the html structure is broken delete div and a

File: templates/product.html [hunk 5]
```diff
@@ -1,42 +1,75 @@
...
-                    <button type="button" class="btn btn-success" >download</button>
-                </a>
-                <button id="share-button" type="button" class="btn btn-success">share</button>
-                <button type="button" class="btn btn-success">rating</button>
-            </div>
-        </div>
-    </div>
-</div>
\ No newline at end of file
+<!DOCTYPE html>
```
Comment (roeeyud) [inline templates/product.html]: Notice that product html shouldn't be a full html file but only the template for the product.

**Comment by roeeyud** [comment]
Remove DS_store files from the PullRequest

**Comment by boazmatz** [comment]
But you closed the Pr

**Pull request closed without merging**